contracts/consensus/Consensus.sol

```diff
@@ -964,6 +1101,17 @@ contract Consensus is MasterCopyNonUpgradable, CoreLifetimeEnum, MosaicVersion,
             _dislocation,
             _metablockHash
         );
+
+        CommitteeI committee = committees[_metablockHash];
+
+        emit MetablockCommitteeFormed(
```

Comment:
I think this event can be emitted from `formCommittee`. We will not need to change signature of startCommittee internal method.

Reply:
Since the committee address is emitted we will need this.
But, I have updated the signature to return the committee address and now the event can be emitted from the function `formCommittee`